ledger/src/blockstore_processor.rs

```diff
@@ -313,6 +319,31 @@ pub fn process_blockstore_from_root(
         genesis_config.operating_mode,
     ));
 
+    {
```

Comment:
new logic here to made `--hard-forks` additive

Reply:
looks nice! :)